freqtrade/optimize/custom_hyperopt.py

```diff
@@ -0,0 +1,153 @@
+# pragma pylint: disable=attribute-defined-outside-init
+
+"""
+This module load custom hyperopts
+"""
+import importlib
+import logging
+import os
+import sys
+from typing import Dict, Any, Callable
+
+from pandas import DataFrame
+
+from freqtrade.constants import Constants
+from freqtrade.optimize.interface import IHyperOpt
+
+sys.path.insert(0, r'../../user_data/hyperopts')
+
+logger = logging.getLogger(__name__)
+
+
+class CustomHyperOpt(object):
+    """
+    This class contains all the logic to load custom hyperopt class
+    """
+    def __init__(self, config: dict = {}) -> None:
+        """
+        Load the custom class from config parameter
+        :param config:
+        :return:
+        """
+
+        # Verify the hyperopt is in the configuration, otherwise fallback to the default hyperopt
+        if 'hyperopt' in config:
```

Comment:
Can be simplified to `hyperopt = config.get('hyperopt') or Constants.DEFAULT_HYPEROPT`